8-code-editor/1-using-a-code-editor/README.md

```diff
@@ -62,18 +63,21 @@ You can also use the command palette. The command palette is an input box where
 
 ![Palette Menu](../images/palette-menu.png)
 
-Once the menu opens, type _open remote repository_, and then select the first option. Multiple repositories that you are part of or that you've opened recently will show up. You can also use a full GitHub URL to select one. Use the following URL and paste into the box:
+Once the menu opens, type _open remote repository_, and then select the first option. Multiple repositories that you are part of or that you've opened recently will show up. You can also use a full GitHub URL to select one.
 
+Copy the url to the repository you just created for your resume site:
+**Replace `your-username` & `repository-name`**
 ```
-https://github.com/microsoft/Web-Dev-For-Beginners
+https://github.com/your-username/repository-name
 ```
 
-✅ If succesful, you'll see all files for this repository loaded in the text editor.
+✅ If successful, you'll see all files for this repository loaded in the text editor.
 
+![Create resume project outline](../images/)
 
 ### 2. Using the URL
 
-You can also use a url directly to load a repository. For example, the full URL for the current repo is [https://github.com/microsoft/Web-Dev-For-Beginners](https://github.com/microsoft/Web-Dev-For-Beginners), but you can swap the GitHub domain with `VSCode.dev/github` and load the repository directly. The resulting URL would be [https://vscode.dev/github/microsoft/Web-Dev-For-Beginners](https://vscode.dev/github/microsoft/Web-Dev-For-Beginners).
+You can also use a url directly to load a repository. For example, the full URL for the resume project repo is https://github.com/your-username/repository-name, but you can swap the GitHub domain with `VSCode.dev/github` and load the repository directly. The resulting URL would be https://vscode.dev/your-username/repository-name

```

Comment:
I don't understand why did you change this line. It was perfect. Please revert the change.